src/states/Cutscene.js

```diff
@@ -1,4 +1,20 @@
-var cutscene = function () {};
+var Cutscene = function () {};
```

Comment:
All of our states are technically objects, that's what the prototype is doing. In either case I think we should stick with this naming.